src/resources/js/views/manager.js

```diff
@@ -310,7 +310,7 @@ tribe.events.views.manager = {};
 
 		var $link = $( this );
 		var url = $link.attr( 'href' );
-		var currentUrl = containerData.url;
+		var currentUrl = containerData.prev_url;
```

Comment:
This makes sense - but for clarity's sake can we change the var name?

```suggestion
		var prevUrl = containerData.prev_url;
```
(and on line 324 where we consume it)

Reply:
@Camwyn 🫡 I've made the update.